servicecontrol/plugins/index.md

```diff
@@ -13,15 +13,32 @@ NOTE: When ServiceControl is introduced into an existing environment the standar
 
 Plugins collect information from NServiceBus and are deployed together with each NServiceBus endpoint. These plugins are optional from the perspective of the NServiceBus framework itself (they are not required by the endpoint), but they are required in order to collect the information that enables ServiceControl (and its clients) to provide the relevant functionality for each plugin.
 
+## Functionalities
 
-## Installing and Deploying
+ServiceControl provide monitoring by analyzing configured audit and fault queues. Based on audit and fault messages it can extract information like: endpoint names, queue names, exception stack traces etc. Based on that information ServiceControl stores them in built-in database and exposes them to 3rd party tools.
+
+Plugins allow to extend given functionalities by providing additional data such as:
+ - [Heartbeat Plugin](heartbeat.md) - report if endpoint has connectivity with ServiceControl by sending heartbeat messages.
+ - [Saga Plugin](saga-audit.md) - report saga related data to allow for better analysis.
+ - [Custom Check Plugin](custom-checks.md) - allow for writing custom checks and sending information about its failure or success.
```

Comment:
>[Custom Check Plugin](custom-checks.md) - allow for writing custom checks and sending information about its failure or success.

[CustomChecks Plugin](custom-checks.md) - Enables custom monitoring abilities for endpoints by allowing the developer to define a set of conditions that needs to be checked. This plugin will report the results of these custom checks to ServiceControl.